src/analyze.cpp

```diff
@@ -8846,13 +8846,13 @@ static void resolve_llvm_types_fn_type(CodeGen *g, ZigType *fn_type) {
     ZigType *gen_return_type;
     if (is_async) {
         gen_return_type = g->builtin_types.entry_void;
-        param_di_types.append(get_llvm_di_type(g, gen_return_type));
+        param_di_types.append(nullptr);
     } else if (!type_has_bits(fn_type_id->return_type)) {
         gen_return_type = g->builtin_types.entry_void;
-        param_di_types.append(get_llvm_di_type(g, gen_return_type));
+        param_di_types.append(nullptr);
```

Comment:
Is this for all 0-bit types? There was talk that `u0` would be respected by gdb (https://sourceware.org/bugzilla/show_bug.cgi?id=25470#c7)

Reply:
As far as the debugger is concerned `u0`/`i0` and `void` return types are both equivalent as the function doesn't return anything.
Let's see if this change fixes the problem observed on Windows binaries.